lighthouse-core/report/html/report-styles.css

```diff
@@ -786,12 +786,13 @@
   border: 0;
 }
 
-.lh-scores-header .lh-gauge__wrapper,
-.lh-scores-header .lh-gauge--pwa__wrapper {
+.lh-scores-header .lh-gauge__wrapper {
   margin: 0 4px;
```

Comment:
Not a huge deal, but do you want to pull this into a variable since the `pwa__wrapper` padding is based on this one? Someday if we change it, it'll be nice if the connection is explicit

Reply:
otherwise could make the comment in `.lh-gauge--pwa__wrapper` more explicit about the 4px matching this one

Reply:
Since this is just a move to padding now, don't think we need a var anymore.  But good advice!

Reply:
💥